compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -577,6 +577,20 @@ public static String cleanupFunctionName(String functionName) {
                 "$" + JVM_RESERVED_CHAR_SET.matcher(functionName).replaceAll("_") : functionName;
     }
 
+    public static boolean isSimpleBasicType(BType bType) {
```

Comment:
If this function is only used in the `JvmPackageGen` class we can move it there. This glass is normally used only if we are writing a function that is used in multiple places

Reply:
Or we can move to TypeTags class

Reply:
I think we already have`isSimpleBasicType` functions maybe it's possible to reuse, let's do this refactoring after beat1.

Reply:
#30639